src/Microsoft.AspNetCore.Testing/project.json

```diff
@@ -15,11 +15,11 @@
     "xunit": "2.1.0"
   },
   "frameworks": {
-    "dnx451": {
+    "net451": {
       "frameworkAssemblies": {
-        "System.Runtime": "",
-        "System.Reflection": "",
-        "System.Threading.Tasks": ""
+        "System.Runtime": {"type": "build" },
```

Comment:
BTW should add a space after the `{` so that the next person editing this won't have to change the formatting again.